event-sourcing/event-repository/event-repository-jpa-liquibase/README.md

```diff
@@ -0,0 +1,13 @@
+# Instructions for creating the database
+
+1. Install Postgres 9.4 or later
+2. Create a user called cppsi
+3. Create a database called cppsi
+4. Run with the following command:
+    mvn resources:resources liquibase:update -Dliquibase.url=jdbc:postgresql://localhost:5432/cppsi -Dliquibase.username=cppsi -Dliquibase.password=password -Dliquibase.logLevel=info
+   Or
+    java -jar envelope-repository-jpa-liquibase-0.1.0-SNAPSHOT.jar --url=jdbc:postgresql://localhost:5432/cppsi --defaultSchemaName=<schema> --username=cppsi --password=password --logLevel=info update
+
+All tables can be dropped by running:
+
+java -jar envelope-repository-jpa-liquibase-0.1.0-SNAPSHOT.jar --url=jdbc:postgresql://localhost:5432/cppsi --username=cppsi --password=password --logLevel=info dropAll
```

Comment:
Jar file name in this readme is wrong, I think. Plus the database-name should probably be changed - eg, event-store